core/base/model/ProjectAliceObject.py

```diff
@@ -78,7 +81,11 @@ def onSessionEnded(self, session): pass
 	def onSay(self, session): pass
 	def onSayFinished(self, session): pass
 	def onSessionQueued(self, session): pass
-	def onMessage(self, intent: str, session): pass
+
+	def onMessage(self, session) -> bool:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4699682)